packages/api/src/routers/external-api/__tests__/alerts.test.ts

```diff
@@ -93,7 +93,7 @@ describe('/api/v1/alerts', () => {
 Array [
   Object {
     "channel": Object {
-      "type": "slack_webhook",
+      "type": "webhook",
```

Comment:
we still want to test `slack_webhook` here to make sure the translator is working fine